### ISSUE TITLE
Cannot import webdriver with selenium > 4.1.0

### ISSUE THREAD

**Issue by benbogart** [wkeeling/selenium-wire#513]
It looks like the selenium release 4.1.1 breaks selenium-wire.  I get the following error when importing webdriver:

```
ImportError                               Traceback (most recent call last)
Input In [1], in <module>
----> 1 from seleniumwire import webdriver

File /opt/homebrew/Caskroom/miniforge/base/envs/selenium/lib/python3.10/site-packages/seleniumwire/webdriver.py:14, in <module>
     12 from selenium.webdriver import FirefoxProfile  # noqa
     13 from selenium.webdriver import Proxy  # noqa
---> 14 from selenium.webdriver import TouchActions  # noqa
     15 from selenium.webdriver import Chrome as _Chrome
     16 from selenium.webdriver import ChromeOptions, DesiredCapabilities

ImportError: cannot import name 'TouchActions' from 'selenium.webdriver' (/opt/homebrew/Caskroom/miniforge/base/envs/selenium/lib/python3.10/site-packages/selenium/webdriver/__init__.py)
```

All is well with selenium 4.1.0.

**Comment by cw-kbra-10887** [comment]
I rolled back to selenium 3.141, I needed it working right away.
I would submit a PR to remove TouchActions, but I don't know the full scope of what would break if I did it

**Comment by nyuszika7h** [comment]
See also: SeleniumHQ/selenium#10395

> I would submit a PR to remove TouchActions, but I don't know the full scope of what would break if I did it

selenium-wire itself does not appear to use TouchActions anywhere, it simply re-exports it, so to avoid breaking any callers importing it from selenium-wire, this should be sufficient:

```py
try:
    from selenium.webdriver import TouchActions
except ImportError:
    pass
```

Then users will only get the error if they're explicitly trying to do `from seleniumwire import TouchActions` with Selenium 4.1.1+, but they would get the same when importing from `selenium.webdriver`.

**Comment by titusfortner** [comment]
There isn't anything that can respond to the TouchActions API any longer, it is completely dead. 

@wkeeling we're going to be removing some additional deprecated classes in the near future, perhaps we can collaborate on it so our projects can stay synchronized? Please reach out to us in the #selenium-tlc channel — https://www.selenium.dev/support/#ChatRoom

**Comment by wkeeling** [comment]
@titusfortner thanks for confirming and also thanks for the heads up about further removals. Yes, lets keep in touch via #selenium-tlc.

**Comment by wkeeling** [comment]
@benbogart thanks for raising the issue. I'll aim to get a fix out ASAP.

**Comment by wkeeling** [comment]
Fixed in Selenium Wire v4.6.2
